#3036 point name in point properties page has a limited visible length

Patrykb0802 opened the pull request:
Added style parameter to point name in point properties view

Patrykb0802 committed:
#3036 point name in point properties page has a limited visible length

Added style parameter to point name in point properties view

Limraj commented on WebContent/WEB-INF/jsp/pointEdit/pointProperties.jsp:
```diff
@@ -49,7 +49,7 @@
       <tr>
         <td class="formLabelRequired"><spring:message code="pointEdit.props.name"/></td>
-        <td class="formField"><input type="text" name="name" value="${status.value}"/></td>
+        <td class="formField" style="width: 98%;"><input type="text" name="name" value="${status.value}" style="width: 98%;"/></td>
```
I suggest using the formLong css class that is used in the description field, and use c:out :

` <td class="formField"><input type="text"` **class="formLong"** ` name="name" value="<c:out value="${status.value}"/>"/></td>`

Patrykb0802 committed:
#3036 point name in point properties page has a limited visible length

Limraj committed:
#3036 point name in point properties page has a limited visible length:
- Created css class formLongFieldSizing;
- Added attribute title and css class formLongFieldSizing in pointProperties.jsp for name fields;
- Set width 60%/40% in dataPointEdit.jsp;
- Use c:out in pointProperties.jsp for name and description fields;

Limraj committed:
#3036 point name in point properties page has a limited visible length:
- Set max-width is 650px
